Merge pull request #8 from nf-core/nf-core-template-merge-2.13.1

lucacozzuto opened the pull request:
Important! Template update for nf-core/tools v2.13.1

<!--
# nf-core/spinningjenny pull request

Many thanks for contributing to nf-core/spinningjenny!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

Remember that PRs should be made against the dev branch, unless you're preparing a pipeline release.

Learn more about contributing: [CONTRIBUTING.md](https://github.com/nf-core/spinningjenny/tree/master/.github/CONTRIBUTING.md)
-->

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/spinningjenny/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/spinningjenny _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nf-test test main.nf.test -profile test,docker`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

lucacozzuto committed:
Merge pull request #8 from nf-core/nf-core-template-merge-2.13.1

Important! Template update for nf-core/tools v2.13.1

ewels committed:
Fix automatic fixes with Prettier

maxulysse commented:
@nf-core-bot fix linting pretty please :pray:

nf-core-bot committed:
[automated] Fix code linting

ewels committed:
Fix indentation whitespace and trailing whitespace

ewels committed:
More manual whitespace fixes for R scripts

lucacozzuto committed:
Merge pull request #10 from nf-core/fix-pre-commit

Fix pre-commit linting

lucacozzuto committed:
several fixes

lucacozzuto committed:
still adapting to new template

lucacozzuto committed:
adding docker files

lucacozzuto committed:
removing fastqc and multiqc

lucacozzuto committed:
linting

lucacozzuto committed:
adding multiqc

mashehu commented on nextflow.config:
```diff
@@ -70,13 +79,13 @@ try {
-// } catch (Exception e) {
-//   System.err.println("WARNING: Could not load nf-core/config/spinningjenny profiles: ${params.custom_config_base}/pipeline/spinningjenny.config")
-// }
+// Load nf-core/rnaseq custom profiles from different institutions.
+try {
+    includeConfig "${params.custom_config_base}/pipeline/rnaseq.config"
+} catch (Exception e) {
+    System.err.println("WARNING: Could not load nf-core/config/rnaseq profiles: ${params.custom_config_base}/pipeline/rnaseq.config")
+}
```
replace `rnaseq` with `spinningjenny`

mashehu commented on nextflow.config:
```diff
@@ -230,18 +242,16 @@ dag {
-    author          = """Nicola Visonà, Luca Cozzuto"""
-    homePage        = 'https://github.com/nf-core/spinningjenny'
-    description     = """simulating the first industrial revolution using Agent Based Models"""
+    name            = 'nf-core/rnaseq'
```
replace with spinningjenny

lucacozzuto committed:
removing rnaseq from nextflow.confi

lucacozzuto committed:
removing multiqc

maxulysse commented:
@nf-core-bot fix linting pretty please :pray:

nf-core-bot committed:
[automated] Fix code linting

nf-core-bot committed:
Remove vulnerable PR-comment artifact pattern

This pipeline's TEMPLATE branch never received the nf-core/tools 4.0.3 sync, so the patch was taken from nf-core/abotyper's TEMPLATE branch with the repository identity in branch.yml rewritten for this pipeline. That identity is the only pipeline-specific content the template renders into any of the patched workflows, so this is the same change this pipeline's own sync would have produced — but the substituted source is worth a look.

These files conflicted with the patch and were reset wholesale to the nf-core/tools 4.0.3 template version, on the basis that they carry no pipeline-specific content. If this branch had local edits in them, those edits are gone — please review: .github/workflows/branch.yml

mashehu committed:
Merge pull request #23 from nf-core/patch

Remove vulnerable PR-comment artifact pattern
